android-core/src/main/java/com/mparticle/modernization/core/MParticleComponent.kt

```diff
@@ -0,0 +1,3 @@
+package com.mparticle.modernization.core
+
+interface MParticleComponent
```

Comment:
Will there be common methods or properties that will be shared between components, or is this simply defining an abstract concept of a `Component`?